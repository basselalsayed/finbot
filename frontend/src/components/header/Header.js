import React from 'react'
import { Link } from 'react-router-dom'

export default function Header() {
  return (
    <header style={headerStyle}>
      <h1>Finbot</h1>
      <Link style={linkStyle} to='/'>Home</Link>
    </header>
  )
}

const headerStyle = {
  background: '#333',
  color: '#fff',
  textAlign: 'center',
  padding: '10px'
}

const linkStyle = {
  color: '#fff',
  textDecoration: 'none'
}