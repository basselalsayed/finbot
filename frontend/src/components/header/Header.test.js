import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import { shallow, mount, render } from 'enzyme';
import Header from './Header';


test('Header renders', () => {
  const component = renderer.create(
    <Router>
      <Header />
    </Router> 
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})

test('Header contains header html element', () => {
  expect(render(<Router><Header /></Router>).text()).toEqual('FinbotHome') 
})