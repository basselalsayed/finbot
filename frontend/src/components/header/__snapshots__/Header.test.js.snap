// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders 1`] = `
<header
  style={
    Object {
      "background": "#333",
      "color": "#fff",
      "padding": "10px",
      "textAlign": "center",
    }
  }
>
  <h1>
    Finbot
  </h1>
  <a
    href="/"
    onClick={[Function]}
    style={
      Object {
        "color": "#fff",
        "textDecoration": "none",
      }
    }
  >
    Home
  </a>
</header>
`;
